components/region_cache_memory_engine/src/background.rs

```diff
@@ -263,6 +290,131 @@ impl BgWorkManager {
 struct BackgroundRunnerCore {
     engine: Arc<RwLock<RangeCacheMemoryEngineCore>>,
     memory_controller: Arc<MemoryController>,
+    range_stats_manager: Option<RangeStatsManager>,
+}
+
+// TODO (afeinberg): Core should just hold Arc<RangeStatsMananger>,
```

Comment:
Forgot to move this to a separate file. May do in a follow up diff to make reviewing easier.

Reply:
Done.